internal/backends/winit/glwindow.rs

```diff
@@ -599,32 +602,36 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WindowAdapterSealed for GLWind
         }
     }
 
-    fn set_position(&self, position: corelib::api::PhysicalPosition) {
+    fn set_position(&self, position: corelib::api::RequestedPosition) {
         match &mut *self.map_state.borrow_mut() {
             GraphicsWindowBackendState::Unmapped { requested_position, .. } => {
                 *requested_position = Some(position)
             }
             GraphicsWindowBackendState::Mapped(mapped_window) => {
+                let physical_position = position.to_physical(self.window().scale_factor());
                 mapped_window.canvas.with_window_handle(|winit_window| {
                     winit_window.set_outer_position(winit::dpi::Position::new(
-                        winit::dpi::PhysicalPosition::new(position.x, position.y),
+                        winit::dpi::PhysicalPosition::new(physical_position.x, physical_position.y),
                     ))
                 })
             }
         }
     }
 
-    fn set_inner_size(&self, size: corelib::api::PhysicalSize) {
+    fn set_size(&self, size: corelib::api::RequestedSize) {
         if let Ok(mut map_state) = self.map_state.try_borrow_mut() {
             // otherwise we are called from the resize event
             match &mut *map_state {
                 GraphicsWindowBackendState::Unmapped { requested_size, .. } => {
                     *requested_size = Some(size)
                 }
                 GraphicsWindowBackendState::Mapped(mapped_window) => {
+                    let physical_size = size.to_physical(self.window().scale_factor());
                     mapped_window.canvas.with_window_handle(|winit_window| {
-                        winit_window
-                            .set_inner_size(winit::dpi::PhysicalSize::new(size.width, size.height));
+                        winit_window.set_inner_size(winit::dpi::PhysicalSize::new(
```

Comment:
here too.

Reply:
Actually here, this might be the correct one since the scale_factor is accurate, so this might not need to be changed.